admin_app_2/src/components/ContentCard.tsx

```diff
@@ -0,0 +1,87 @@
+import ContentViewModal from "@/components/ContentModal";
+import { appColors, appStyles, sizes } from "@/utils";
+import { Edit } from "@mui/icons-material";
+import { Button, Card, Typography } from "@mui/material";
+import Link from "next/link";
+import React from "react";
+import { Layout } from "./Layout";
+
+const ContentCard = ({
```

Comment:
This should mirror the wireframe - we should agree on any changes there and not deviate from it in implementation unless it's necessary. I also made some adhoc changes in my PR too... I'll align it in my next PR.

Reply:
I've aligned a bunch but going to leave content text in. Titles are not sufficient to tell the difference between content.